pkg/helmexec/exec.go

```diff
@@ -84,6 +86,20 @@ func getHelmVersion(helmBinary string, runner Runner) (semver.Version, error) {
 	return parseHelmVersion(string(outBytes))
 }
 
+func GetPluginVersion(name, pluginsDir string) (*semver.Version, error) {
+	plugins, err := plugin.FindPlugins(pluginsDir)
+	if err != nil {
+		return nil, err
+	}
+	for _, plugin := range plugins {
+		if plugin.Metadata.Name == name {
+			return semver.MustParse(plugin.Metadata.Version), nil
```

Comment:
semver.MustParse will panic. why not use semver.Parse?

Reply:
Yes, it looks like `NewVersion` is better in the method.